src/frontend/src/lib/components/dapps/DappPromoBanner.svelte

```diff
@@ -0,0 +1,40 @@
+<script lang="ts">
+	import Img from '$lib/components/ui/Img.svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+	import type { DApp } from '$lib/types/dapp';
+	import { replacePlaceholders } from '$lib/utils/i18n.utils';
+
+	export let dApp: DApp;
+</script>
+
+<article class="relative flex h-64 items-end overflow-hidden rounded-2xl">
+	{#if dApp.screenshots && dApp.screenshots.length > 0}
+		<div class="absolute">
+			<Img src={dApp.screenshots[0]} alt={dApp.name} />
```

Comment:
Isn't a screenshot a must to be showcased in the promo banner?

Reply:
Yeah that's a good point, I'll add a separate property to make it more clear

Reply:
I honestly did not understood that the property you added made it clear. Also not sure that's the appropriate solution.

For this PR maybe let's make this obvious on the type of the property and let's iterate in another PR.

I did not got it. Should rather be the type of `Dapp` be improved to make this requirement obvious?

```
export let dapp: Omit<Dapp, "background"> & Required<Pick<Dapp, "background">>
```

WDYT?